freud/diffraction.pyx

```diff
@@ -0,0 +1,348 @@
+import numpy as np
```

Comment:
Remove `diffraction.py` and put its functionality into `diffraction.pyx` or else the imports will get confused.